### ISSUE TITLE
fix error while not on remote debugging

### ISSUE THREAD

**Issue by imranariffin** [hooligram/hooligram-client#23]
Currently will get error if not on remote debugging

**Comment by imranariffin** [comment]
While working #24 , I found the offending line:
```
// app/screens/OnBoardingRequestCode/presenter.js
const countryCode = _countryCode.replace(/[^0-9])/, '')
// note the unpaired closing parentheses
```
@cglotr

**Comment by cglotr** [comment]
@imranariffin are you including the fix in #24?

**Comment by imranariffin** [comment]
Nope
